rest-api/src/main/resources/paths/participant_files/v3_participants_self_files.yml

```diff
@@ -0,0 +1,20 @@
+get:
+    operationId: getParticipantFiles
+    summary: Get a list of participant files from the user
+    tags:
+        - Participant Files
+        - _For Consented Users
+    security:
+        - BridgeSecurity: []
+    parameters:
+        - $ref: ../../index.yml#/parameters/offsetKey
+        - $ref: ../../index.yml#/parameters/pageSize
+    responses:
+        200:
+            description: OK
+            schema:
+                $ref: ../../definitions/paged_resources/participant_file.yml
+        401:
+            $ref: ../../responses/401.yml
+        403:
+            $ref: ../../responses/403_not_developer.yml
```

Comment:
Instead of 403, what we want is
```
        410:
            $ref: ../../responses/410.yml
        412:
            $ref: ../../responses/412.yml
```

Same for the other APIs.